app/frontend/src/router/minesOperatorScreening.js

```diff
@@ -1,77 +1,82 @@
 /**
  * Mines Operator Screening Form Routes
  */
+import { FormNames } from '@/utils/constants';
+
+const baseName = 'MinesOperatorScreening';
```

Comment:
Shouldn't all these constants be with FormNames?  Seems like they all relate to form/route naming...

Reply:
the intent for basename is for internal router component naming. At this time though, I stuck with keeping things working instead of trying to do consolidation onto the FormNames constants - the only real functional difference between them is that baseName is PascalCase and FormNames is all lowercase. Something we can look into later if there's more time/desire for cleanup...